registry.k8s.io/images/k8s-staging-cluster-api/images.yaml

```diff
@@ -447,6 +449,7 @@
     "sha256:7d72254cb2b1f3d535447be7973bbe00b5160ecf8edc6c42e44d7e1e13714808": ["v1.10.2"]
     "sha256:73adc0fdac0495c94cc911c7dacb3871faa1f13429f383f035a28fa225a402c9": ["v1.10.3"]
     "sha256:9550f704d4bff78a476cece45b6e4dfe2c63e244036ea76b5d06f7de28a379e3": ["v1.10.4"]
+    "sha256:695045c27cc7062d4731b7f1e47b7395a81e19202bcf42a479960794d8d52499": ["v1.11.0"]
```

Comment:
SHA `695045c27cc7` matches kubeadm-bootstrap-controller v1.11.0 at the [staging repo](https://console.cloud.google.com/gcr/images/k8s-staging-cluster-api/global/kubeadm-bootstrap-controller@sha256:695045c27cc7062d4731b7f1e47b7395a81e19202bcf42a479960794d8d52499/details?inv=1&invt=Ab56Hw).